core/dbt/compilation.py

```diff
@@ -54,6 +54,24 @@ def print_compile_stats(stats):
     logger.info("Found {}".format(stat_line))
 
 
+def _node_enabled(node):
+    # Disabled models are already excluded from the manifest
+    if node.resource_type == NodeType.Test and not node.config.enabled:
+        return False
+    else:
+        return True
+
+
+def _generate_stats(manifest):
+    stats = defaultdict(int)
+    for node_name, node in itertools.chain(
+            manifest.nodes.items(),
+            manifest.macros.items()):
+        if _node_enabled(node): stats[node.resource_type] += 1
```

Comment:
looks like the `flake8` styleguide checker we use isn't happy about this line:
```
core/dbt/compilation.py:70:31: E701 multiple statements on one line (colon)
```

via https://circleci.com/gh/fishtown-analytics/dbt/19687?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Can you just split this out onto two lines?

Reply:
Sure - thanks